.github/workflows/tests.yml

```diff
@@ -244,13 +248,14 @@ jobs:
         bin/setup_daq
     - name: Loading docker images
       run: |
-        bin/load_images faucet faux setup base
+        bin/load_images faucet faux setup base subset
     - name: Running device_coupler test
       env:
         DOCKER_STARTUP_TIMEOUT_MS: 60000
         GCP_BASE64_CRED: ${{ secrets.GCP_BASE64_CRED }}
         GCP_REFLECT_KEY_BASE64: ${{ secrets.GCP_REFLECT_KEY_BASE64 }}
       run: |
+        python3 -m unittest discover -s device_coupler/testing/unit -p "test_*.py"
```

Comment:
unless there's a really good reason for it, all the testing stuff should be under testing/ (not device_coupler/testing) -- then this line itself shouldn't be necessary since it's covered under the regular slate of unit tests.

Reply:
Relocated unit tests